spec/civiltime.html

```diff
@@ -1,230 +1,558 @@
+        1. Let _minute_ be _greater_.[[Minute]] - _smaller_.[[Minute]].
+        1. Let _second_ be _greater_.[[Second]] - _smaller_.[[Second]].
+        1. Let _millisecond_ be _greater_.[[Millisecond]] - _smaller_.[[Millisecond]].
+        1. Let _microsecond_ be _greater_.[[Microsecond]] - _smaller_.[[Microsecond]].
+        1. Let _nanosecond_ be _greater_.[[Nanosecond]] - _smaller_.[[Nanosecond]].
+        1. <mark>TODO: Balance.</mark>
+        1. Return CreateDuration(0, 0, 0, _hour_, _minute_, _second_, _millisecond_, _microsecond_, _nanosecond_).
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause>
+      <h1>Temporal.Time.prototype.withDate ( _date_ [ , _disambiguation_ ] )</h1>
+      <p>
+        The `withDate` method takes two arguments, _date_ and _disambiguation_.
+        The following steps are taken:
+      </p>
+      <emu-alg>
+        1. Let _time_ be the *this* value.
+        1. If Type(_time_) is not Object or _time_ does not have an [[InitializedTemporalTime]] internal slot, then
+          1. Throw a *TypeError* exception.
+        1. Let _date_ be ? ToDate(_date_).
+        1. If _disambiguation_ is not *undefined*, then
+          1. Set _disambiguation_ to ? ToString(_disambiguation_).
+        1. Else
+          1. Set _disambiguation_ to `"constrain"`.
+        1. Return CreateDateTime(_date_.[[Year]], _date_.[[Month]], _date_.[[Day]],
+          _time_.[[Hour]], _time_.[[Minute]], _time_.[[Second]],
+          _time_.[[Millisecond]], _time_.[[Microsecond]], _time_.[[Nanosecond]]).
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause>
+      <h1>Temporal.Time.prototype.toString ( )</h1>
+      <p>
+        The following steps are taken:
+      </p>
+      <emu-alg>
+        1. Let _time_ be the *this* value.
+        1. If Type(_time_) is not Object or _time_ does not have an [[InitializedTemporalTime]] internal slot, then
+          1. Throw a *TypeError* exception.
+        1. Let _hour_ be _time_.[[Hour]] formatted as a two-digit decimal number, padded to the left with a zero if necessary.
+        1. Let _minute_ be _time_.[[Minute]] formatted as a two-digit decimal number, padded to the left with a zero if necessary.
+        1. Let _seconds_ be ! FormatSecondsStringPart(_time_.[[Second]], _time_.[[Millisecond]], _time_.[[Microsecond]], _time_.[[Nanosecond]]).
+        1. Return the string-concatenation of _hour_, the code unit 0x003A (COLON), _minute_, and _seconds_.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause>
+      <h1>Temporal.Time.prototype.toLocaleString ( …_args_ )</h1>
+      <p>
+        The following steps are taken:
+      </p>
+      <emu-alg>
+        1. Let _time_ be the *this* value.
+        1. If Type(_time_) is not Object or _time_ does not have an [[InitializedTemporalTime]] internal slot, then
+          1. Throw a *TypeError* exception.
+        1. Let _dateFormat_ be ? Construct(%DateTimeFormat%, _args_).
```

Comment:
is this intrinsic from intl? Intl is not necessarily available to 262 engines.

Reply:
Yes. We can copy some wording from 262 to make this sorta optional.